examples/3d/fog.rs

```diff
@@ -114,8 +117,7 @@ fn setup_pyramid_scene(
     commands.spawn(PointLightBundle {
         transform: Transform::from_xyz(0.0, 1.0, 0.0),
         point_light: PointLight {
-            intensity: 300_000.,
-            range: 100.,
+            intensity: 10_000.,
```

Comment:
This light is quite dim now and maybe it's making it difficult to see what the sphere is trying to show in this example? Although I'm not sure what exactly that would be...